etc/config/c++.defaults.properties

```diff
@@ -27,7 +27,7 @@ group.clang.compilers=clang7:clangdefault
 group.clang.intelAsm=-mllvm --x86-asm-syntax=intel
 compiler.clang7.exe=/usr/bin/clang++-7
 compiler.clang7.name=clang 7
-compiler.clangdefault.exe=/usr/bin/clang++
```

Comment:
Can you revert this change? If you need to set this locally, I'd recommend creating a copy of this file called `c++.local.properties` which overrides every other properties present

Reply:
Sure, Will remove this

Reply:
Please do :) It's still modified in the version on github.